src/test/scala/net/manub/embeddedkafka/EmbeddedKafkaSpecSupport.scala

```diff
@@ -17,22 +17,22 @@ abstract class EmbeddedKafkaSpecSupport extends TestKit(ActorSystem("embedded-ka
 
   def kafkaIsAvailable(kafkaPort: Int = 6001): Unit = {
     system.actorOf(TcpClient.props(new InetSocketAddress("localhost", kafkaPort), testActor))
-    expectMsg(1 second, ConnectionSuccessful)
+    expectMsg(ConnectionSuccessful)
```

Comment:
I didi it because I had intermittent local failures with the 1s timeout, with the default value these problems disappeared. Will change it back to 1 second.